include/sys/time.h

```diff
@@ -163,8 +173,28 @@ struct itimerval {
 
 #ifdef _KERNEL
 
+#include <sys/callout.h>
+
 typedef struct proc proc_t;
 
+/* Kernel interval timer. */
+typedef struct {
+  timeval_t
+    kit_next; /* absolute time of nearest expiration, 0 means inactive */
```

Comment:
Could you write type and name in one line?

Reply:
Done.